aspnet/migration/rc1-to-rc2.rst

```diff
@@ -136,7 +193,7 @@ HttpPlatformModule
 
 ``Microsoft.AspNetCore.IISPlatformHandler`` is now ``Microsoft.AspNetCore.Server.IISIntegration``.
 
-HttpPlatformModule was replaced by ASP.NET Core Module. The web.config created by the Publish IIS tool now configures IIS to use ASP.NET Core Module instead of HttpPlatformHandler to reverse-proxy requests to Kestrel.
+HttpPlatformModule was replaced by ASP.NET Core Module. The ``web.config`` created by the Publish IIS tool now configures IIS to use ASP.NET Core Module instead of HttpPlatformHandler to reverse-proxy requests to Kestrel.
 
 The code snippet below shows how to configure the new Publish IIS tool in ``project.json`` file:
```

Comment:
Add wrapping curly braces to the code-block below so that the JSON will highlight correctly and not generate build warnings on the latest version of Sphinx.